models/job.js

```diff
@@ -41,7 +42,10 @@ const MODEL = {
         .boolean()
         .description('Flag if the job is archived')
         .example(true)
-        .default(false)
+        .default(false),
+
+    annotations: Annotations.annotations
```

Comment:
Ahh I didn't realize adding something to models added it to the database. Will remove.